### ISSUE TITLE
Why are router stack up pages?

### ISSUE THREAD

**Issue by renatoholanda** [bencompton/framework7-react#52]
When I call other page from the menu on sidebar (left panel), it will render a new page component, ignoring the cached page, and then I have this case:

```
<div class="views"></div>
    <div class="view view-main navbar-fixed navbar-through" id="main-view" data-page="home">
        <div class="pages" data-vue-ref="[object Object]">
            <div class="page default-bg-color page-on-center" data-page="home" id="home">...</div>
            <div class="page default-bg-color page-on-center" data-page="fast-delivery" id="fast-delivery">...</div>
        </div>
    </div>
</div>
```

I have this code in the menu:

```
<ListButton view="#main-view" href="/" animatePages={false}>
    <IconHome />
</ListButton>
<ListButton view="#main-view" href="/fast-delivery" animatePages={false}>
    <IconFastDelivery />
</ListButton>
```

I need to open some pages without "back" functionality. Open new page withou caching. Is it a bug or configuration?

**Comment by bencompton** [comment]
From your code, the behavior I would expect is that you click one of the links, and that new page will be added to the existing pages in the DOM cache, but there will not be an animated transition to the new page.

Is your goal to have the new page not animate and also replace the last page in the DOM cache? If so, I think the `force` option on the link might be what you want:

http://framework7.io/react/link.html

**Comment by renatoholanda** [comment]
> Is your goal to have the new page not animate and also replace the last page in the DOM cache?
Exactly. Your solution works fine and don`t stack up all pages, but first page is always rendered (home). Why?

My mainview it's:
```
<View id="main-view" navbarThrough dynamicNavbar={true} main>
   <Pages>
      <Home />
   </Pages>
</View>
```

I think if I remove <Home /> and load it dynamically, it will solve my problem, but I don`t found how make this.

If I put url="/" the <Link href="/"> doesn't work. No erros, but no load page.
